src/combination.jl

```diff
@@ -252,3 +250,32 @@ function plot_order_temporalCombin(params1, params2, g1s, g2s, named1, named2)
         title = string(named1, " --> ", named2, " - ", named2, " --> ", named1),
     )
 end
+
+####--------------- Loewe additivity ------------------####
+""" Find the inverse of a hill function (concentration), given the parameters and the effect. """
+function inv_hill(p::Array{Float64, 1}, y)
+    #p = [EC50, min, max, steepness], y:effect. it returns concentration
+    conc = p[1] * (((y - p[2]) / (p[3] - y)) ^ p[4])
+    return conc
+end
+
+""" The combination Index for Loewe. """
+function loewe(d1, p1, d2, p2, conc1, conc2)
+    # x: the combined effect
+    f(x) = (d1 / inv_hill(p1, x)) + (d2 / inv_hill(p2, x)) - 1.0
+
+    combined_effect = find_zero(f, [(minimum([p1[2], p2[2]]) + eps()), (maximum([p1[3], p2[3]])-eps())])
+    return combined_effect
+end
+
+""" Apply Loewe to a range of concentrations. """
+function apply_loewe(params1, params2, conc1, conc2)
+
+    combined_effects = zeros(length(conc1), length(conc2))
+    for (ind1, d1) in enumerate(conc1)
+        for (ind2, d2) in enumerate(conc2) # keep d1 constant and loop over d2
+            combined_effects[ind1, ind2] = loewe(d1, params1, d2, params2, conc1, conc2)
+        end
+    end
+    return combined_effects
+end
```

Comment:
Please ignore this for now, it just loops over all the concentrations to find all the combinations.